.ci/chromium-setup.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+set -e
+# this script install chromium headless for a debian system used in CI.
+snap install chromium
```

Comment:
Doesn't adding 

```
addons:
  chrome: stable
```

fix that?

Reply:
@vitoravelino i'm trying to use `chromium` which is more updated atm.

Anywyas is kind wip, i need to find out the repository of latest chromium for ubuntu so it's compatible with the latest chrome-driver :racehorse:

Reply:
thx @vitoravelino anyways for that suggestion

Reply:
No problem. :wink: